ts/DataGrid/DataGrid2/Actions/ColumnsResizer.ts

```diff
@@ -0,0 +1,250 @@
+/* *
+ *
+ *  Data Grid Columns Resizer class.
+ *
+ *  (c) 2020-2024 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Dawid Dragula
+ *  - Sebastian Bochan
+ *
+ * */
+
+'use strict';
+
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import DataGridTable from '../DataGridTable.js';
+import DataGridColumn from '../DataGridColumn.js';
+
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+/**
+ * The class that handles the resizing of columns in the data grid.
+ */
+class ColumnsResizer {
+
+    /* *
+    *
+    *  Properties
+    *
+    * */
+
+    /**
+     * The minimum width of a column.
+     */
+    private static readonly MIN_COLUMN_WIDTH = 20;
```

Comment:
do we need it as static?

Reply:
This is a global setting and I think static fields fit better for this. There is no need to copy this value for each instance of the class.